roles/build-image/templates/foreman-proxy/Dockerfile.j2

```diff
@@ -0,0 +1,21 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install http://yum.theforeman.org/releases/{{ version }}/el7/x86_64/foreman-release.rpm && \
+    yum -y install foreman-release-scl ansible && \
+    yum -y install foreman-proxy {{ rpms_to_install | join(' ') }} && \
+    yum clean all
+
+COPY container-assets/logs.yml /etc/foreman-proxy/settings.d/logs.yml
+COPY container-assets/settings.yml /etc/foreman-proxy/settings.yml
+
+{% for config in configs_to_copy %}
+  COPY container-assets/{{config}} /etc/foreman-proxy/settings.d/{{config}}
+{% endfor %}
```

Comment:
Does each `COPY` create a new layer? If so, wouldn't it be better to generate a tar with all the configs on the builder's side and then `ADD` the archive, having docker uncompress it as a single layer?

Reply:
Fair point, there is definitely room for making this more optimized.

Reply:
The [best practices guide](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) discourages using `ADD` and even states that multiple files should be added one by one rather than all at once. I left it as it is for now, but I can switch if we decide `ADD` is really what we want.

Reply:
/me see that page for the first time

for some reason I recall some doc saying that less layers beats anything else, but guess it is outdated by now. nevermind then